spec/examples.txt

```diff
@@ -0,0 +1,23 @@
+example_id                                          | status | run_time        |
+--------------------------------------------------- | ------ | --------------- |
+./spec/features/searching_spec.rb[1:1:1]            | passed | 0.00047 seconds |
+./spec/features/searching_spec.rb[1:2:1]            | passed | 0.0007 seconds  |
+./spec/features/searching_spec.rb[1:2:2]            | passed | 0.00043 seconds |
+./spec/features/searching_spec.rb[1:2:3]            | passed | 0.00132 seconds |
+./spec/features/searching_spec.rb[1:3:1]            | passed | 0.02642 seconds |
+./spec/features/welcome_spec.rb[1:1]                | passed | 0.02191 seconds |
+./spec/features/welcome_spec.rb[1:2]                | passed | 7.6 seconds     |
+./spec/helpers/display_format_helper_spec.rb[1:1:1] | passed | 0.00374 seconds |
+./spec/helpers/display_format_helper_spec.rb[1:1:2] | passed | 0.00124 seconds |
+./spec/helpers/display_format_helper_spec.rb[1:1:3] | passed | 0.00132 seconds |
+./spec/helpers/display_format_helper_spec.rb[1:2:1] | passed | 0.00227 seconds |
+./spec/helpers/display_format_helper_spec.rb[1:2:2] | passed | 0.00165 seconds |
+./spec/helpers/display_format_helper_spec.rb[1:2:3] | passed | 0.00292 seconds |
+./spec/helpers/display_format_helper_spec.rb[1:3:1] | passed | 0.00143 seconds |
+./spec/helpers/display_format_helper_spec.rb[1:3:2] | passed | 0.00195 seconds |
+./spec/helpers/display_format_helper_spec.rb[1:3:3] | passed | 0.00199 seconds |
+./spec/helpers/result_helper_spec.rb[1:1:1]         | passed | 0.00137 seconds |
+./spec/helpers/result_helper_spec.rb[1:2:1]         | passed | 0.0015 seconds  |
+./spec/lib/searching/results_spec.rb[1:1:1]         | passed | 0.00101 seconds |
+./spec/lib/searching/results_spec.rb[1:2:1]         | passed | 0.00094 seconds |
+./spec/lib/searching/results_spec.rb[1:3:1]         | passed | 0.00094 seconds |
```

Comment:
this file should be added to the `.gitignore` file for the app and not be committed